src/nginxconfig/i18n/en/templates/domain_sections/php.js

```diff
@@ -39,4 +39,19 @@ export default {
     enableMagentoRules: `${common.enable} ${common.magento}-specific rules`,
     joomlaRules: `${common.joomla} rules`,
     enableJoomlaRules: `${common.enable} ${common.joomla}-specific rules`,
+    phpServer: `${common.php} server`,
+    phpMustBeEnabledOnOneSite: `${common.php} must be enabled on at least one site to configure global ${common.php} settings.`,
```

Comment:
```suggestion
```
I don't think this string is needed any more, as this was the message shown in the global PHP tab when no sites had PHP?